ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpClientPlugin.kt

```diff
@@ -32,24 +32,18 @@ public interface HttpClientPlugin<out TConfig : Any, TPlugin : Any> {
 }
 
 /**
- * Try to get the [plugin] installed in this client. Returns `null` if the plugin was not previously installed.
+ * Returns a [plugin] installed in this client. Returns `null` if the plugin was not previously installed.
  */
 public fun <B : Any, F : Any> HttpClient.plugin(plugin: HttpClientPlugin<B, F>): F? =
```

Comment:
Does it make sense to break it in 2.0 and do `plugin` and `pluginOrNull` naming?
same naming strategy is used in server plugins

Reply:
+1

Reply:
done